### DISCUSSION TITLE
How to respond to comments on the Embark repo?

### DISCUSSION THREAD

**Discussion by tombh** [Rust-GPU/rust-gpu#42] [Q&A] [answered]
I'm following a few threads on the old Embark repo and sometimes see users unaware that the repo is deprecated. Should we respond to them, pointing to the new repo here? I know it's been decided not to migrate the Embark repo and all its issues/PRs (see https://github.com/Rust-GPU/rust-gpu/issues/6), so has there been discussion about other ways to migrate? Say for example dumping the threads from Embark as plain text in the first comment of new issues here?

**Comment by utensil** [comment]
Tools like https://github.com/mattduck/gh2md could also facilitate the process.

**Comment by schell** [comment, marked as answer]
I think it's fine to comment on those threads and link to a new issue here as needed. Are there any issues in particular that you'd like to see mirrored here @tombh?

**Comment by tombh** [comment]
With @schell's help I've written and run a script to migrate all the open Embark issues. However Github disabled the user, @rust-gpu-bot, that I used to create the new issues. I filed a ticket last week to reinstate it, but they're busier than normal, so it may take a while yet for the issues to appear in this repo.

**Reply by LegNeato** [reply]
Wow, awesome!
